pkg/operators/machine-api-operator.go

```diff
@@ -66,6 +66,10 @@ var _ = Describe("[Feature:Operators] Machine API operator deployment should", f
 		Expect(framework.IsDeploymentAvailable(client, maoManagedDeployment, framework.MachineAPINamespace)).To(BeTrue())
 
 	})
+})
+
```

Comment:
can you help me understand what this is solving? All the IT validations within this describe still run in parallel right?

Reply:
Can you share CI failures/flakes this PR would solve?

Reply:
- https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/184/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-aws-operator/1290284947541594112  
- https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/184/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-gcp-operator/1290284947608702976
- https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/180/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-azure-operator/1286272804777365504
- Guess this one as well: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/180/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-gcp-operator/1286272804827697152